cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/TaskWorkerHttpHandlerInternal.java

```diff
@@ -55,7 +56,7 @@
   @Inject
   public TaskWorkerHttpHandlerInternal(CConfiguration cConf, Consumer<String> stopper) {
```

Comment:
Constructor should be non public if it is annotated with `@Inject`.

Reply:
Done